llvm/docs/CommandGuide/llvm-objcopy.rst

```diff
@@ -464,6 +464,18 @@ them.
  Read a list of symbols from <filename> and change their visibility to the
  specified value. Visibility values: default, internal, hidden, protected.
 
+.. option:: --skip-symbol <symbol>
+
+ Do not change parameters of symbol <symbol> when executing other options that
+ can change the symbol's name, binding or visibility.
+
+.. option:: --skip-symbols <filename>
+
+ Read a list of symbols from <filename> and run as if --skip-symbol=<symbol>
+ is set for each one. <filename> contains one symbol per line and may contain
```

Comment:
Please add backticks around `<filename>` in the same manner as other similar references in the doc. Similarly, please change the markdown for the `--skip-symbol` option reference in this text, so that it provides the auto link that you get with other option references.